### ISSUE TITLE
Feature request openstack_networking_port_v2: Allow using security group names instead of only IDs

### ISSUE THREAD

**Issue by johanneskastl** [terraform-providers/terraform-provider-openstack#976]
**Feature Request for openstack_networking_port_v2**:

Allow using names for security groups instead of only allowing the IDs. This would make code a lot easier to read, as `ssh` is easier to understand than `1234-abcd`.

### Terraform Version
```
$ terraform version
Terraform v0.12.23
+ provider.openstack v1.26.0
```

### Affected Resource(s)
`openstack_networking_port_v2`

### Terraform Configuration Files
This is the currently valid configuration, see [the documentation](https://www.terraform.io/docs/providers/openstack/r/networking_port_v2.html#security_group_ids)

```hcl
resource "openstack_networking_port_v2" "create_ports" {
  name               = "somename"
  network_id         = "1234-5678"
  security_group_ids = ['1234-abcd', '5678-efgh']
}
```

**Comment by kayrus** [comment]
@johanneskastl I don't think it is a good idea, while terraform operates with resources based on their ID's, not names. We already have a problem with nova based API, which actually allows to operate with resources by their [names](https://www.terraform.io/docs/providers/openstack/r/compute_instance_v2.html#name-1) and this gives a lot of headache, especially when the network is renamed. I'd recommend to use IDs.
